openedx_events/learning/data.py

```diff
@@ -257,3 +257,42 @@ class UserNotificationData:
     app_name = attr.ib(type=str)
     course_key = attr.ib(type=CourseKey)
     context = attr.ib(type=dict, factory=dict)
+
+
+@attr.s(frozen=True)
+class ProgramData:
+    """
+    Attributes defined for the Open edX Program data object.
+
+    Arguments:
+        uuid (str): The UUID of the program (from Course-Discovery)
+        title (str): The title of the program
+        program_type (str): The type slug of the program (e.g. professional, microbachelors, micromasters, etc.)
```

Comment:
Same comment as [L289](https://github.com/openedx/openedx-events/pull/253/files#r1276540953). Possible values are defined... I wanna say in course-discovery?? 😭

Reply:
They are and they are synced to Credentials as part of the `copy_catalog` workflow. We have access to them all in Credentials. I'm not sure where they are defined in Discovery but I can add a link to them if I find it if you'd like!